### PR TITLE
pythonXXPackages.httpserver: init at 1.1.0

### PR DESCRIPTION
**Pull request by P-E-Meunier**
###### Motivation for this change
SimpleHTTPServer is a python package that starts an HTTP server serving files in the local directory.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -2964,6 +2964,33 @@ in {
     };
   };
 
+  httpserver = buildPythonPackage rec {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: please move expression to `pkgs/development/python-modules/httpserver/default.nix`

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -2964,6 +2964,33 @@ in {
...
+    name = "${pname}-${version}";
+    pname = "httpserver";
+    version = "1.1.0";
+    buildInputs = with self; [ docopt ];
+    doCheck = false;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: why do you disable the tests? If you disable the tests, always include a comment.

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -2964,6 +2964,33 @@ in {
...
+    src = pkgs.fetchurl {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: fetchPypi

File: pkgs/top-level/python-packages.nix [hunk 4]
```diff
@@ -22154,6 +22181,7 @@ in {
     };
   };
 
+
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: this shouldn't be included

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by FRidh** [comment]
Don't merge, rebase instead. If you need help with it or like me to do it let me know.

**Comment by P-E-Meunier** [comment]
Please go ahead!

If this PR requires me to start fiddling with git, I'm willing to abandon this it.

**Comment by FRidh** [comment]
there it is https://github.com/NixOS/nixpkgs/commit/5db271c99a3313d36d72e008ad5db1335024de5e

**Comment by P-E-Meunier** [comment]
Awesome! Thanks a lot!

**Pull request closed without merging**